simulator/host_datastore_system.go

```diff
@@ -108,6 +108,7 @@ func (dss *HostDatastoreSystem) CreateLocalDatastore(c *types.CreateLocalDatasto
 	}
 
 	ds.Summary.Type = string(types.HostFileSystemVolumeFileSystemTypeOTHER)
+	ds.Summary.MaintenanceMode = "normal"
```

Comment:
Let's use the generated constant for this value: `types.DatastoreSummaryMaintenanceModeStateNormal`